test_integration/devchains.sh

```diff
@@ -3,20 +3,16 @@
 operation=$1
 originChain="dev-origin"
 auxiliaryChain="dev-auxiliary"
-auxChainIdentifier="1000"
 
 function start_chains {
+
 echo starting $originChain chain
 ./node_modules/.bin/mosaic start $originChain
-./node_modules/.bin/mosaic subgraph $originChain $auxChainIdentifier origin  http://localhost:9535 http://localhost:6516
```

Comment:
Why it's removed?

Reply:
because we moved sub-graph deployment code to seperate files. (test_integration/scripts/eip20_token/deploy_subgraph.sh)